lms/templates/inline_analytics.html

```diff
@@ -65,7 +65,10 @@
           <div class="grid-wrapper">
             <p>
               <strong class="num-students"></strong> ${_('students answered this question:')}<br/>
-              <strong class="num-students-extra"></strong>
+              <strong class="num-students-extra-first"></strong>${_('correct and')}
+              <strong class="num-students-extra-first-continued"></strong>${_('incorrect (First Attempt).')}<br/>
+              <strong class="num-students-extra-last"></strong>${_('correct and')}
+              <strong class="num-students-extra-last-continued"></strong>${_('incorrect (Last Attempt).')}<br/>
```

Comment:
It would be nice to name the classes num-students-extra-first-correct and num-students-extra-first-incorrect for readability.